doc/man/8/rbd-nbd.rst

```diff
@@ -18,9 +18,9 @@ Synopsis
 Description
 ===========
 
-**rbd-nbd** is a client for RADOS block device (rbd) images like rbd kernel module.
-It will map a rbd image to a nbd (Network Block Device) device, allowing access it
-as regular local block device.
+**rbd-nbd** is a client for RADOS block device (rbd) images, like the rbd kernel module.
```

Comment:
s/like/for example/

Reply:
"like" seems appropriate to me here.  "for example" would change the meaning -- it is saying that `rbd-nbd` is _similar to_ the rbd kernel module.